### PR TITLE
*: update pd-client

### PR DESCRIPTION
**Pull request by overvenus**
pingcap/pd#206

Please merge pingcap/pd#213 frist, I will update Godeps.json later.

### CODE REVIEW

File: store/tikv/ticlient_test.go [hunk 1]
```diff
@@ -23,10 +23,9 @@ import (
 )
 
 var (
-	withTiKV     = flag.Bool("with-tikv", false, "run tests with TiKV cluster started. (not use the mock server)")
-	etcdAddrs    = flag.String("etcd-addrs", "127.0.0.1:2379", "etcd addrs")
-	pdLeaderPath = flag.String("pd-path", "/pd", "PD leader path on etcd")
-	clusterID    = flag.Int("cluster", 1, "cluster ID")
+	withTiKV  = flag.Bool("with-tikv", false, "run tests with TiKV cluster started. (not use the mock server)")
+	etcdAddrs = flag.String("etcd-addrs", "127.0.0.1:2379", "etcd addrs")
```
Comment (siddontang) [inline store/tikv/ticlient_test.go]: pdAddrs

**Comment by shenli** [comment]
You can use [this script](https://github.com/pingcap/tidb/blob/master/updatedep.sh) to update repo in the vendor. It will update Godeps.json automatically.

**Comment by qiuyesuifeng** [comment]
@shenli  This script  is only for kvproto package, what he want is the master pd-client commit,  it is better to update Godeps.json instead of using branch commit.

File: store/tikv/ticlient_test.go (hunk 1 above)
Comment (qiuyesuifeng) [inline store/tikv/ticlient_test.go]: +1

File: store/tikv/ticlient_test.go [hunk 2]
```diff
@@ -36,7 +35,7 @@ func newTestStore(c *C) *tikvStore {
 
 	if *withTiKV {
 		var d Driver
-		store, err := d.Open(fmt.Sprintf("tikv://%s%s?cluster=%d", *etcdAddrs, *pdLeaderPath, *clusterID))
+		store, err := d.Open(fmt.Sprintf("tikv://%s/?cluster=%d", *pdAddrs, *clusterID))
```
Comment (qiuyesuifeng) [inline store/tikv/ticlient_test.go]: Seems we can remove `/`.
@siddontang  how about you?

Comment (siddontang) [inline store/tikv/ticlient_test.go]: +1

**Comment by overvenus** [comment]
PTAL @qiuyesuifeng @shenli @siddontang

**Comment by shenli** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

File: store/tikv/kv.go [hunk 3]
```diff
@@ -45,22 +45,22 @@ type Driver struct {
 }
 
 // Open opens or creates an TiKV storage with given path.
-// Path example: tikv://etcd-node1:port,etcd-node2:port/pd-path?cluster=1
+// Path example: tikv://etcd-node1:port,etcd-node2:port/?cluster=1
```
Comment (siddontang) [inline store/tikv/kv.go]: fix comment

**Pull request merged**